blacktie/integration-tests/src/test/java/org/jboss/narayana/blacktie/jatmibroker/xatmi/CSControl.java

```diff
@@ -53,10 +54,14 @@ public void tearDown() {
         try {
             if (server != null) {
                 log.debug("destroying server process");
-                server.interrupt();
-                server.getProcess().destroy();
-                log.debug("destroyed server process");
+                try {
+                    int pid = getUnixPID(server.getProcess());
+                    Runtime.getRuntime().exec("kill -INT " + pid).waitFor();
```

Comment:
cs.exe works both in server and client.
runServer() to start a server process with the command "cs.exe -s testsui -p 12342 -i sid" 
runTest(name) to start a client process with the command "cs.exe name"

It needs to shutdown the server by sending the SIGINT to the server process. In term of windows which has not the signal, it looks like need "Ctrl+Break" .